### PR TITLE
Release ouroboros-network

### PR DESCRIPTION
**Pull request by coot**
# Description

Prepare to release `ouroboros-network-0.8.0.0`.

# Checklist

- Branch
    - [ ] Updated changelog files.
    - [ ] Commit sequence broadly makes sense
    - [ ] Commits have useful messages
    - [ ] The documentation has been properly updated
    - [ ] New tests are added if needed and existing tests are updated
    - [ ] If serialization changes, user-facing consequences (e.g. replay from genesis) are confirmed to be intentional.
- Pull Request
    - [ ] Self-reviewed the diff
    - [ ] Useful pull request description at least containing the following information:
      - What does this PR change?
      - Why these changes were needed?
      - How does this affect downstream repositories and/or end-users?
      - Which ticket does this PR close (if any)? If it does, is it [linked](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)?
    - [ ] Reviewer requested

### CODE REVIEW

File: cardano-ping/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Revision history for cardano-ping
 
+## 0.2.0.0 -- 2023-05-26
```
Comment (bolt12) [inline cardano-ping/CHANGELOG.md:3]:
```suggestion
## 0.2.0.1 -- 2023-05-26
```

**Review by bolt12 — APPROVED** [review]
What about https://github.com/input-output-hk/ouroboros-network/pull/4395?

**Comment by bolt12** [comment]
I approved but still needs changes, please note that. I should have asked for changes instead @coot

**Comment by coot** [comment]
I don't want to include #4395 in `0.8`.

**Pull request merged**